test/test_vim.py

```diff
@@ -63,8 +64,9 @@ def test_strwidth():
 @with_setup(setup=cleanup)
 def test_chdir():
     pwd = vim.eval('getcwd()')
-    vim.chdir('/')
```

Comment:
Thanks to @equalsraf and @jamessan we found the missing bit of Windows-specific handling.

From vim:
ttps://github.com/vim/vim/blob/0b9e4d1224522791c0dbbd45742cbd688be823f3/src/misc2.c#L5445

Which should belong in Neovim's:
https://github.com/neovim/neovim/blob/master/src/nvim/file_search.c#L1404 

Pending commit to Neovim's #810.